json/src/spec/engine.rs

```diff
@@ -91,5 +93,22 @@ mod tests {
             Engine::Tendermint(_) => {} // Tendermint is unit tested in its own file.
             _ => panic!(),
         };
+
+        let s = r#"{
+			"cuckoo": {
```

Comment:
Fix indentation.

Reply:
Other test codes in same file has similar problem. I'll patch this in master branch manually.